src/ZendeskApi.Client/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1,5 +1,8 @@
+using System;
+using System.Net.Http;
 using Microsoft.Extensions.DependencyInjection;
 using ZendeskApi.Client.Options;
+#pragma warning disable 618
 
 namespace ZendeskApi.Client.Extensions
```

Comment:
Can we remove the Extensions namespace please?

Reply:
Do we want to do that, it will break consumers implementations if they're already using these extensions? It's been in this namespace for 2 years https://github.com/justeat/ZendeskApiClient/commit/30cfb9a10c39f886853c6af7d63863593eadaee5#diff-7d9f67410130ec9e68e6f5eb87ae63bbR4

Reply:
Well I always considered it a bit of a bad practice to use that namespace. 

But yeah - I guess if you don't want to break exciting clients, maybe leave as is - I should have paid more attention when it was changed 2 years ago LOL